robottelo/ssh.py

```diff
@@ -73,18 +76,28 @@ def _get_connection(
         hostname = settings.server.hostname
     if username is None:
         username = settings.server.ssh_username
-    if key_filename is None:
+    if key_filename is None and password is None:
         key_filename = settings.server.ssh_key
+        # If we don't have key even in settings, read password instead:
+        if key_filename is None:
+            password = settings.server.ssh_password
```

Comment:
it's another part. Here I wanted to read `password` setting from property file only in case key is not available neither in passed parameters nor in file. It's not about priority, it's about doing less operations:)